src/fields.cpp

```diff
@@ -457,14 +457,14 @@ bool fields_chunk::alloc_f(component c) {
           component bc = direction_component(Bx, component_direction(c));
           if (!f[bc][cmp]) {
             f[bc][cmp] = new realnum[gv.ntot()];
-            for (size_t i = 0; i < gv.ntot(); i++)
+            PS1LOOP_OVER_VOL(gv, c, i)
               f[bc][cmp][i] = 0.0;
```

Comment:
It seems like overkill to thread these loops.

Reply:
I think the idea was to allocate arrays using the same threads that would loop over them while stepping (to avoid things similar to false sharing). I didn't notice any real speedup from this, though.